pilot/pkg/bootstrap/server.go

```diff
@@ -226,6 +226,7 @@ func NewServer(args *PilotArgs) (*Server, error) {
 	// common https server for webhooks (e.g. injection, validation)
 	s.initHTTPSWebhookServer(args)
 
+	args.Config.ControllerOptions.CABundlePath = s.caBundlePath
```

Comment:
Can you point me why we need this field?

Reply:
we will use it in pilot/pkg/serviceregistry/kube/controller/multicluster.go:L144 :L145 to patch the webhook's caBundle

Reply:
@hzxuzhonghu caBundle is required for remote webhook configuration to talk to the webhook server.